server/gc/src/main/java/org/apache/accumulo/gc/SimpleGarbageCollector.java

```diff
@@ -172,6 +176,7 @@ public static void main(String[] args) throws UnknownHostException, IOException
     }
   }
 
+
```

Comment:
This line will probably be deleted by the formatter. Please run the formatter before merging.